mongo_connector/doc_managers/neo4j_doc_manager.py

```diff
@@ -53,7 +52,26 @@ def upsert(self, doc, namespace, timestamp):
         "ns": namespace,
         "_ts": timestamp
     }
+    doc = self._formatter.format_document(doc)
+    self.store_nodes_and_relationships(doc, doc_type, doc_id)
+    
   
+  def store_nodes_and_relationships(self, doc, doc_type, doc_id):
+    tx = self.graph.cypher.begin()
+    parameters = {'id':doc_id}
+    for key in doc.keys():
+      LOG.error(type(doc[key]))
+      if (type(doc[key]) is str):
+        value = doc[key]
+      else:
+        value = ""
+      parameters.update({ key: value })
+    parameters = ', '.join("{!s}:{!r}".format(k,v) for (k,v) in parameters.items())
+    query = "CREATE (c:Document:{doc_type} {{ {parameters} }}) RETURN c".format(doc_type=doc_type, parameters=parameters)
```

Comment:
don't do this, always use parameters within the statement no string formatting (except for the label)